Adds the Pharmacist

Bribrooo opened the pull request:
## About the PR
Adds the Pharmacist role, a chemist who focuses on roleplay interactions, RP-meds, and others.
Works with the psychologist, chemists, or independently.
Has access to Chemistry and Psychologist.
Adds news access Pharmacist. Requires to have played 24h as Chemist.

## Why / Balance
With the addition of more and more reagents, the chemists are often overwhelmed with requests, adding this role not only encourages chemistry players to finally become social (totally not like me), but can also prescribe the RP medication without approval of the CMO, still requiring it for any other reagent.

Spawn point, role, inventory, signs, description, icon, access, airlocks, closet, and stamp were already made and tested successfully.

It can probably be mapped independently or in station medical outposts since it is its own booth.

**TO DO**
- [x] More drip options
- [x] Discrete belt chem bag
- [x] Pharmacy prescription paper to fix
- [ ] SoP regulations
- [x] Guidebook entries
     - [x] Pharmacist
     - [x] Chemicals/Medicine

**Reagents to do**
- [x] Asparatine - mild painkiller
     - [x] Sodium Salicylate
          - [x] Methyl Salicylate
               - [x] Salicylic Acid
                    - [x] Sodium Phenolate (used for carbonylation)
                         - [x] Benzenesulfonic Acid (used for endothermic reaction)
               - [x] Methanol
     - [x] Acetyl Chloride
          - [x] Acetic Anhydride
          - [x] Hydrogen Chloride
          - [x] (Hydrochloric Acid - made when Hydrogen Chlorine enters in contact with water.) 
     - [x] Acetic Acid - used to make more acetic anhydride (methanol + acetic acid (+ sulfuric acid) = acetic anhydride)
- [X] Vicodillicaine - strong headache med, hallucinogen
- [X] Zepprocane - barbiturate, used as anxiolytic and anticonvulsant
    - [X] Barbituric Acid - an acidic reagent, reacts with artifacts
         - [X] Hydrogen Iodide - light molecule often used in pharmaceutics and barbituric acid reaction
         - [X] Sodium amalgam - from an exothermic reaction causing an alloy made of mercury and sodium which happens 
          naturally when they're pressed together. causes no damage (0.0001 x 15 dmg)
- [x] Cordivol - "α1- + β-selective" α1 antagonist, non-selective: β1 + β2 antagonist
     - [x] Propylene glycol - used in foods
          - [x] Propylene oxide - flammable
          - [x] Aryloxypropanolamine - a precursor to most β-blockers
               - [x] Catechol - from catechin, but not here though
                    - [x] Hydrogen Peroxide - oxidizing agent
               - [x] Epichlorohydrin - synthesis-related molecule
                    - [x] Allyl Chloride - used to be an antiseptic
- [x] Naguuprofen - mild headache med, normally used to combat hangovers. isobutylphenylethanol is mixed with carbon and oxygen alongside hydrogen chlorine, with iron acting as a catalyst
     - [x] Isobutylphenylethanol - from the former and hydrogen
          - [x] Isobutylacetophenone - from a mix of the two former
               - [x] Isobutylbenzene - from toluene and propene, with a sodium-potassium catalyst
               - [x] Hydrogen Fluoride - hydrogen and fluorine with heat applied
                    - [x] Tuolene - mix of benzene and methanol, using sulfuric acid as a catalyst
                    - [x] Propene - from acetone, charcoal, and some heat
  
<details><summary>Abandonned/Scrapped</summary>
Dawcicadone - cholinesterase inhibitors and glutamate regulators? memory loss. Vanlicylicane - Mild leg pain painkiller. Grahatrocine - light painkiller for shoulder pain. Scotycoine - mild painkiller for stomach pain.
</details> 


## Media
![image](https://github.com/user-attachments/assets/87330603-8d4e-43d8-bed7-fd3d1f887359)
![image](https://github.com/user-attachments/assets/6379fc58-0e42-4f9e-a0c3-d2b3a54060a8)
![image](https://github.com/user-attachments/assets/ba5ce691-58ed-4572-9d32-372919ffc2d6)
![image](https://github.com/user-attachments/assets/82e20517-e469-4320-b31a-887d31631132)
https://github.com/user-attachments/assets/f99adbf1-8a04-418a-b5a3-3430ee373384



- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
- add: A new employee arrives at the station... it's the Pharmacist!
- add: A whole new plethora of pharmaceutical reagents and their predecessors!

Bribrooo committed:
Adds Vicodilicaine

Adds vicodilicaine, genesis of the pharmacology...

Bribrooo committed:
Moves the _CD folder to DeltaV folder

Moves the _CD folder to DeltaV folder as commit 336ae81 requested, also to avoid confusion

Bribrooo committed:
Zepprocane & More!

Adds Zepprocane, Barbituric Acid, Sodium Amalgam, Hydrogen Iodide, and future chem for Asparatine

Bribrooo committed:
Pharmacist clothing + Asparatin

Adds a lab coat and a suit for the pharmacist.
Adds Asparatin, at least its effects (thx for the help solidus YES IM TAGGING YOU I WAS DESPERATE)

Co-Authored-By: VMSolidus <16548818+VMSolidus@users.noreply.github.com>

Bribrooo committed:
Adds the Pharmacist Job

muahahaha it works muahahaha

Bribrooo committed:
FTL touch up

Bribrooo commented:
Add any name, recipe, and effects of rp medications you'd like to see in-game for your character(s) or general use!

Avalon-Proto commented:
I hope you don't plan to stick them in the same room as the chemists. As that can get rather crowded already

Bribrooo commented:
READ WHAT I WROTE

Avalon-Proto commented:
... Forget I said anything. Jedi mindtrick bullshittery. I missed that last section :blunt:

IamVelcroboy commented:
I just skimmed this and I noticed the prescriptions, something I've wanted to add for a long time. I think it would be a massive RP booster for medical/chem/patients.

Wondering if we need to give doctors stamps, or a stamp function in their PDA, or even a stamp they can eject from the PDA. Something to verify a script that the patient can take to the pharmacy to get filled and it's *probably* not a forgery

DangerRevolution commented:
shouldn't the reagents be in a seperate PR?

Bribrooo committed:
Modified the prescription paper

Bribrooo commented:
I wouldn't add the reagents only for roleplay to put more pressure on the chemists. That's why I didn't choose to put the reagents in another PR.

Timemaster99 commented:
The outfit looks far too similar to the senior physician, and with its current relation to medical, will cause player confusion. Adding a new trim color to the white coat could be a better option.

I'll also bring attention to the major issue, which is how similar this role is to Chemist. Roles for the game need to be designed in a way that is easy to understand at a glance, because the description strings and PR text will almost never be read by a majority of players. I don't see anything in the PR that would stop the chemist from doing their job, or would stop the pharmacist from being an extra chemist. In the current state, I can see Pharmacist acting as a chemist in med for a majority of the shift, because med outposts are lonely and the chemists will likely have many of the component compounds of the RP meds already made.

I do think there are some solutions to this, both of which can be implemented together. 
First would be to implement the smartfridge from ss13, which could create a sort of storefront design for over-the-counter medications, and then add a pharmacist access to it. From there, the pharmacist could be represented in a way that's more akin to a store-keep, and stock the medications of people who come to visit. 
The second solution is to add symptoms in-game, either randomly-acquired or chosen as a trait (similar to the old runny nose trait). These would be harmless, or perhaps inflict meaningless damage, which would give more players reasons to roleplay sickness and visit the pharmacy for their symptoms. Perhaps, they'll even stock up for future symptoms if they choose to have migraines as a trait.

Aside from these issues, I do enjoy the role and the RP it could present. Perhaps doctors and psychologists can gain access to these prescription papers to write them for patients, pending pharmacist approval

Colin-Tel commented:
Personally, I don't really see why'd you add a pharmacist when you could just add another chemist to a station.

Bribrooo commented:
> The outfit looks far too similar to the senior physician, and with its current relation to medical, will cause player confusion. Adding a new trim color to the white coat could be a better option.

I tried to toy with colors unsuccessfully, I'm still toying with it but it's more placeholder colors than anything for now

> I can see  Pharmacist acting as a chemist in med for a majority of the shift, because med outposts are lonely and the chemists will likely have many of the component compounds of the RP meds already made.

I'm introducing new side reagents that will take a long process to make a decent amount or obtain them simply. Also, a lot of those rp reagents will be designed to be a pain in the ass to make to either test your chemistry skills or so that it can simply feel high standards since pharmacology is an unforgiving sector of medicine (trust me on this one).

> From there, the pharmacist could be represented in a way that's more akin to a store-keep, and stock the medications of people who come to visit.

That's what I'm trying to do; by making them separate from the main medical wing, like the psychiatrist, so they are their own booth that they can run independently, ready to help people with questions about health, or hand out their prescriptions.

> Personally, I don't really see why'd you add a pharmacist when you could just add another chemist to a station.

I feel like adding another chemist to the station wouldn't add the more in-depth roleplay people could have, by having a person whose purpose is to get them their medication, and possibly also hook them up with a psychologist (which is deserted most of the time)

Bribrooo committed:
modified the coat and suit

modified the colors from light blue to light green

Bribrooo committed:
modified the job marker, pda, stamp and door

changed the light blue to light green

Bribrooo committed:
added discrete pharmacist chem bag

added discrete pharmacist chem bag + fixed pharmacistshirt.rsi

Bribrooo committed:
Added the chembag to basic loadout

Bribrooo committed:
updates rp chem group

Catanautical commented:
> I feel like adding another chemist to the station wouldn't add the more in-depth roleplay people could have, by having a person whose purpose is to get them their medication, and possibly also hook them up with a psychologist (which is deserted most of the time)

I do very agree with the fact that the pharmacist should be the linking factor of roleplay between the chemists, the psychiatrist, and medical, but it could also include being called a dispensary for perscription drugs such as cannabis and tobacco, and also work with the new Interdyne chemists you see occasionally to get more legal-skirting drugs. This way all of these become options that the pharmacy/dispensary can be involved in. All of this stuff needs to be webbed in to have as much interaction with the crew on the station as possible during their job, that way it's actually fulfilling to play and enjoyable to the rest of the crew. But, this is all my two-cents.


> Personally, I don't really see why'd you add a pharmacist when you could just add another chemist to a station.

And the problem here with adding 'just another chemist' is because they're literally 'JUST ANOTHER CHEMIST'. A chemist has no obligation to interact with any crew whatsoever, and if you just add another chemist, they will just complain that the other chemists have already made all the medical drugs and have nothing to do, since it is almost entirely the chemist culture to only focus on medbay and that's it. We're trying to create a job similar in nature that does not have obligation to dispense to medbay other than the psychologist and/or psyciatrist so that they can focus on the rest of the crew's drug use.

Bribrooo committed:
Adds Asparatine and the whole process behind it (12 new reagents)

Adds Asparatine, Sodium Salicylate, Acetyl Chloride, Acetic Acid, Methyl Salicylate, Salicylic Acid, Methanol,  Sodium Phenolate, Benzenesulforic Acid, Acetic Anhydride, Hydrogen Chloride and Hydrochloric Acid

Bribrooo committed:
Fixed FATL bugs

Bribrooo committed:
Fixed even more FATL bugs

Bribrooo committed:
Fixed bugs again

Bribrooo committed:
Fixed a bug...

Bribrooo commented:
Still didnt fix the current bug

Bribrooo committed:
Finally fixes bugs and whatnot

Bribrooo committed:
Merge branch 'DeltaV-Station:master' into pharmacology

Bribrooo committed:
Adds Naguuprofen (and 6 new reagents)

And all of that bugs-free!

Bribrooo committed:
Added Guidebook Entry

Bribrooo committed:
Adds a jumpsuit and jumpskirt alternative

Bribrooo committed:
Update identification_cards.yml

Bribrooo committed:
Update loadout-groups.ftl

Bribrooo committed:
Update identification_cards.yml

Bribrooo committed:
Merge branch 'master' into pharmacology

Signed-off-by: BriBrooo <45323530+Bribrooo@users.noreply.github.com>

Bribrooo committed:
Merge branch 'master' into pharmacology

Bribrooo committed:
Prepares future β-blockers reagents + Sodium Amalgam spark SFX

Adds Aryloxypropanolamine, Epichlorohydrin, AllylChloride, Catechol, HydrogenPeroxide and modifies the sparks of the sodium amalgam effect.

Bribrooo committed:
Adds Propene glycol and propene oxide

and everything bug free too

Bribrooo committed:
Adds Cordivol and fixes the guidebook (i think?)

bug free too!!

Bribrooo committed:
forgot to save last file open...

Bribrooo committed:
Adds Debivolol + β-blocker touch ups

Bribrooo commented:
I will need the SoP regulations worked with @Timemaster99 and then it should be totally ready for a merge if there's nothing wrong with the current thing. But probably a lot to do still maybe.

Bribrooo committed:
Update pharmacy.yml

Bribrooo committed:
Update pharmacy.yml

Bribrooo committed:
Merge branch 'master' into pharmacology

Signed-off-by: BriBrooo <45323530+Bribrooo@users.noreply.github.com>

Bribrooo committed:
bug fixes

Bribrooo committed:
Merge branch 'pharmacology' of https://github.com/Bribrooo/Delta-V-HA into pharmacology

Bribrooo committed:
bug fixes

Bribrooo committed:
Merge branch 'master' into pharmacology

Bribrooo committed:
bug fixes requiem

Bribrooo committed:
Update pharmachemicals.yml

Bribrooo committed:
Update pharmacy.yml

Bribrooo committed:
fixes (?)

Bribrooo committed:
fixes (?) chap. 2 uncertitudes extended

Bribrooo committed:
I want to eat myself

Bribrooo committed:
Merge branch 'master' into pharmacology

Signed-off-by: BriBrooo <45323530+Bribrooo@users.noreply.github.com>

Bribrooo committed:
loadout fixes

Bribrooo committed:
Merge branch 'pharmacology' of https://github.com/Bribrooo/Delta-V-HA into pharmacology

Bribrooo committed:
Update role_loadouts.yml

Bribrooo committed:
trying to fix and not break my house

Bribrooo committed:
Update pharmacist.yml

Bribrooo committed:
Update pharmacist.yml

Bribrooo committed:
Update pharmacist.yml

Bribrooo committed:
Update pharmacist.yml

Bribrooo committed:
Update identification_cards.yml

Bribrooo committed:
tried fix

Bribrooo committed:
Update pharmachemicals.yml

Bribrooo committed:
Merge branch 'master' into pharmacology

DangerRevolution commented on Resources/Locale/en-US/_CD/reagents/meta/physical-desc.ftl:
Why was this removed?

DangerRevolution commented on Resources/Locale/en-US/deltav/job/job-description.ftl:
```diff
@@ -4,3 +4,4 @@ job-description-clerk = Organize trials, notarize documents, review charges, and
 job-description-prosecutor = Take statements from security and prepare cases against those accused of commiting crimes.
 job-description-courier = Deliver mail and other packages from and to logistics. Avoid dogs.
 job-description-security-borg = Purpose-built to ensure the right of every crew member to liberty, justice and freedom, ensure the peace aboard the space station by following your laws and patrolling the halls.
+job-description-pharmacist = A pharmaceutical engineer, focused on the manufacturing and prescription of pharmaceutical reagents on the station.
```
```suggestion
job-description-pharmacist = A pharmaceutical engineer, focused on the manufacturing and prescription of pharmaceutical reagents on the station.

```

DangerRevolution commented on Resources/Locale/en-US/deltav/job/job-names.ftl:
```diff
@@ -13,3 +14,4 @@ JobChiefJustice = Chief Justice
 JobClerk = Clerk
 JobProsecutor = Prosecutor
 JobSecurityBorg = Security Cyborg
+JobPharmacist = Pharmacist
```
```suggestion
JobPharmacist = Pharmacist

```

DangerRevolution commented on Resources/Locale/en-US/deltav/reagents/meta/pharmacy.ftl:
```diff
@@ -26,4 +26,25 @@ reagent-name-soretizone = Soretizone
+reagent-desc-cordivol = An α1 antagonist, non-selective β-blocker. Used in severe cases of systolic heart failure. No real issues with mixing other drugs, alcohol is safe.
+
+reagent-name-debivolol = X-Debivolol
+reagent-desc-debivolol = placeholder script.
```
```suggestion
reagent-desc-debivolol = placeholder script.

```
And finish this.

DangerRevolution commented on Resources/Locale/en-US/deltav/reagents/meta/physical-desc.ftl:
```diff
@@ -0,0 +1,3 @@
+reagent-physical-desc-bending = light bending
+reagent-physical-desc-sharp = sharp
+reagent-physical-desc-solidlauryl = white and solid
```
```suggestion
reagent-physical-desc-solidlauryl = white and solid

```

DangerRevolution commented on Resources/Locale/en-US/deltav/reagents/pharmacymsg.ftl:
_CD content should stay in _CD folders.

DangerRevolution commented on Resources/Locale/en-US/deltav/reagents/pharmacymsg.ftl:
```diff
@@ -68,22 +70,64 @@ reagent-effect-medaddiction-6 = You can feel your hands shaking.
 reagent-effect-painkiller-normal3 = Your entire body goes numb briefly.
 reagent-effect-painkiller-normal4 = You feel your pain fade somewhat.
 
+reagent-effect-painkiller-mild1 = Your body hurts a bit less.
```
Why does mild come before normal?

DangerRevolution commented on Resources/Locale/en-US/deltav/reagents/sodiumamalgam.ftl:
```diff
@@ -0,0 +1 @@
+sodiumamalgam-spark = The sodium melts with the mercury!
```
```suggestion
sodiumamalgam-spark = The sodium melts with the mercury!

```

DangerRevolution commented on Resources/Locale/en-US/deltav/reagents/pharmacymsg.ftl:
```diff
@@ -68,22 +70,64 @@ reagent-effect-medaddiction-6 = You can feel your hands shaking.
+reagent-effect-cordivol-debivolol-mix2 = You feel like you're about to die...
+reagent-effect-cordivol-debivolol-mix3 = I need air... Fresh air...
+reagent-effect-cordivol-debivolol-mix4 = Can't... Chest...
+reagent-effect-cordivol-debivolol-mix5 = My... heart...
```
```suggestion
reagent-effect-cordivol-debivolol-mix5 = My... heart...

```

DangerRevolution commented on Resources/Prototypes/DeltaV/Access/medical.yml:
```diff
@@ -1,3 +1,7 @@
+
+- type: accessLevel
+  id: Pharmacist
+  name: id-card-access-level-pharmacist
```
```suggestion
  name: id-card-access-level-pharmacist
  
```

DangerRevolution commented on Resources/Prototypes/DeltaV/Entities/Clothing/Uniforms/jumpskirts.yml:
```diff
@@ -127,3 +127,13 @@
 
+- type: entity
+  parent: ClothingUniformSkirtBase
+  id: ClothingUniformJumpskirtPharma
```
Don't use Pharma; annoying to find it if you try type in Pharmacist

DangerRevolution commented on Resources/Prototypes/DeltaV/Entities/Clothing/Uniforms/jumpskirts.yml:
```diff
@@ -127,3 +127,13 @@
+  - type: Sprite
+    sprite: DeltaV/Clothing/Uniforms/Jumpskirt/pharmacist.rsi
+  - type: Clothing
+    sprite: DeltaV/Clothing/Uniforms/Jumpskirt/pharmacist.rsi
```
```suggestion
    sprite: DeltaV/Clothing/Uniforms/Jumpskirt/pharmacist.rsi

```

DangerRevolution commented on Resources/Prototypes/DeltaV/Entities/Clothing/Uniforms/jumpsuits.yml:
```diff
@@ -321,3 +321,24 @@
 
+- type: entity
+  parent: ClothingUniformBase
+  id: ClothingUniformJumpsuitPharmaShirt
```
Don't use Pharma; annoying to find it if you try type in Pharmacist

DangerRevolution commented on Resources/Prototypes/DeltaV/Entities/Clothing/Uniforms/jumpsuits.yml:
```diff
@@ -321,3 +321,24 @@
+
+- type: entity
+  parent: ClothingUniformBase
+  id: ClothingUniformJumpsuitPharma
```
Don't use Pharma; annoying to find it if you try type in Pharmacist

DangerRevolution commented on Resources/Prototypes/DeltaV/Entities/Objects/Misc/paper.yml:
```diff
@@ -30,3 +30,29 @@
+- type: entity
+  name: pharmacy prescription
+  parent: Paper
+  id: PaperPrecsription
```
spelling mistake consider your PR over

DangerRevolution commented on Resources/Prototypes/DeltaV/Entities/Objects/Misc/paper.yml:
```diff
@@ -30,3 +30,29 @@
+  name: pharmacy prescription
+  parent: Paper
+  id: PaperPrecsription
+  description: 'A piece of paper to justify the reagents you detain on your person. Issued by the pharmacist.'
```
```suggestion
  description: 'A piece of paper to justify the reagents you carry on your person. Issued by the pharmacist.'
```
detain is a weird word

DangerRevolution commented on Resources/Prototypes/DeltaV/Entities/Objects/Misc/rubber_stamp.yml:
```diff
@@ -11,6 +11,20 @@
+  name: pharmacist's rubber stamp
+  parent: RubberStampBase
+  id: RubberStampPharmacist
+  suffix: DO NOT MAP
```
```suggestion
  suffix: DO NOT MAP
  noSpawn: true
```
DO NOT MAP should be set to noSpawn to avoid it being mapped

DangerRevolution replied:
Worth considering leaving this stamp in the pharmacist office in case there isn't one aboard; so prescriptions can be officially stamped.

DangerRevolution commented on Resources/Prototypes/DeltaV/Entities/Objects/Specific/Chemistry/chem_bag_pharmacist.yml:
```diff
@@ -0,0 +1,33 @@
+        - PillCanister
+        - Bottle
+        - Syringe
+        - HandLabeler
```
> A bag for storing pharmacy products, such as pills, pill canisters, bottles, and syringes.

> HandLabeler

DangerRevolution commented on Resources/Prototypes/DeltaV/Entities/Objects/Specific/Chemistry/chem_bag_pharmacist.yml:
```diff
@@ -0,0 +1,33 @@
+  name: pharmacist bag
+  id: ChemBagPharmacist
+  parent: BaseStorageItem
+  description: A bag for storing pharmacy products, such as pills, pill canisters, bottles, and syringes.
```
This is a copy of the Chemistry bag; so should be parented as such.

DangerRevolution commented on Resources/Prototypes/DeltaV/Entities/Structures/Doors/Windoors/windoor.yml:
```diff
@@ -162,3 +162,12 @@
+  components:
+  - type: ContainerFill
+    containers:
+      board: [ DoorElectronicsPharmacist ]
```
```suggestion
      board: [ DoorElectronicsPharmacist ]

```

DangerRevolution commented on Resources/Prototypes/DeltaV/Entities/Structures/Wallmounts/Signs/signs.yml:
```diff
@@ -38,4 +38,12 @@
+  parent: BaseSignDirectional
+  id: SignDirectionalPharmacy
+  name: pharmaceutic lab sign
+  description:  A direction sign, pointing out which way the pharmceutic lab is.
```
```suggestion
  description:  A direction sign, pointing out which way the Pharmacy is.
```

DangerRevolution commented on Resources/Prototypes/DeltaV/Guidebook/medical.yml:
```diff
@@ -0,0 +1,4 @@
+- type: guideEntry
+  id: Pharmacist
+  name: guide-entry-pharmacist
+  text: "/ServerInfo/Guidebook/DeltaV/Medical/Pharmacist.xml"
```
```suggestion
  text: "/ServerInfo/Guidebook/DeltaV/Medical/Pharmacist.xml"

```

DangerRevolution commented on Resources/Prototypes/DeltaV/Reagents/pharmacy.yml:
Stop stealing _CD files.

DangerRevolution commented on Resources/ServerInfo/Guidebook/DeltaV/Medical/Pharmacist.xml:
Why will this be the only guidebook designed for a single role rather than a concept?

Even Chemistry is just a guidebook for chemicals; and not for the role.

None of Engineering guidebooks are for the roles; it's for different engines or gameplay concepts.

None of Securitys' guidebooks are a "how to play secoff" but rather a general guide on Security department and different other breakdowns

Bribrooo replied:
I added the entry because I thought either:
A. Every job would have their basic wiki equivalent inside of it
B. I can always change the name to Pharmacy | Pharmaceutics | Pharmaceutical Lab | ...

TadJohnson00 commented:
I don't see the point of this role when you could, as Colin said before, just add an extra chemist slot to the station. You can add another chemistry workstation to some maps as well, if it's necessary. I'd be open to further discussion but right now this role has too much of an overlap with chemists to warrant its own role, stamp, etc.

Bribrooo committed:
Update Resources/Locale/en-US/deltav/job/job-description.ftl

Co-authored-by: Danger Revolution! <142105406+DangerRevolution@users.noreply.github.com>
Signed-off-by: BriBrooo <45323530+Bribrooo@users.noreply.github.com>

Bribrooo committed:
Update Resources/Locale/en-US/deltav/job/job-names.ftl

Co-authored-by: Danger Revolution! <142105406+DangerRevolution@users.noreply.github.com>
Signed-off-by: BriBrooo <45323530+Bribrooo@users.noreply.github.com>

Bribrooo committed:
Update Resources/Locale/en-US/deltav/reagents/meta/pharmacy.ftl

Co-authored-by: Danger Revolution! <142105406+DangerRevolution@users.noreply.github.com>
Signed-off-by: BriBrooo <45323530+Bribrooo@users.noreply.github.com>

Bribrooo committed:
Update Resources/Locale/en-US/deltav/reagents/meta/physical-desc.ftl

Co-authored-by: Danger Revolution! <142105406+DangerRevolution@users.noreply.github.com>
Signed-off-by: BriBrooo <45323530+Bribrooo@users.noreply.github.com>

Bribrooo committed:
Update Resources/Locale/en-US/deltav/reagents/sodiumamalgam.ftl

Co-authored-by: Danger Revolution! <142105406+DangerRevolution@users.noreply.github.com>
Signed-off-by: BriBrooo <45323530+Bribrooo@users.noreply.github.com>

Bribrooo committed:
Update Resources/Prototypes/DeltaV/Entities/Structures/Doors/Windoors/windoor.yml

Co-authored-by: Danger Revolution! <142105406+DangerRevolution@users.noreply.github.com>
Signed-off-by: BriBrooo <45323530+Bribrooo@users.noreply.github.com>

Bribrooo committed:
Update Resources/Prototypes/DeltaV/Entities/Structures/Wallmounts/Signs/signs.yml

Co-authored-by: Danger Revolution! <142105406+DangerRevolution@users.noreply.github.com>
Signed-off-by: BriBrooo <45323530+Bribrooo@users.noreply.github.com>

Bribrooo committed:
Update Resources/Prototypes/DeltaV/Guidebook/medical.yml

Co-authored-by: Danger Revolution! <142105406+DangerRevolution@users.noreply.github.com>
Signed-off-by: BriBrooo <45323530+Bribrooo@users.noreply.github.com>

Bribrooo committed:
Update Resources/Locale/en-US/deltav/reagents/pharmacymsg.ftl

Co-authored-by: Danger Revolution! <142105406+DangerRevolution@users.noreply.github.com>
Signed-off-by: BriBrooo <45323530+Bribrooo@users.noreply.github.com>

Bribrooo committed:
Update Resources/Prototypes/DeltaV/Access/medical.yml

Co-authored-by: Danger Revolution! <142105406+DangerRevolution@users.noreply.github.com>
Signed-off-by: BriBrooo <45323530+Bribrooo@users.noreply.github.com>

Bribrooo committed:
Update Resources/Prototypes/DeltaV/Entities/Clothing/Uniforms/jumpskirts.yml

Co-authored-by: Danger Revolution! <142105406+DangerRevolution@users.noreply.github.com>
Signed-off-by: BriBrooo <45323530+Bribrooo@users.noreply.github.com>

Bribrooo committed:
Update Resources/Prototypes/DeltaV/Entities/Objects/Misc/paper.yml

Co-authored-by: Danger Revolution! <142105406+DangerRevolution@users.noreply.github.com>
Signed-off-by: BriBrooo <45323530+Bribrooo@users.noreply.github.com>

Bribrooo committed:
Update Resources/Prototypes/DeltaV/Entities/Objects/Misc/rubber_stamp.yml

Co-authored-by: Danger Revolution! <142105406+DangerRevolution@users.noreply.github.com>
Signed-off-by: BriBrooo <45323530+Bribrooo@users.noreply.github.com>

ThataKat commented:
I have NO experience with contrib. stuff, but maybe this could be an alternate job title for Chemists in the mean time if the content doesn't make sense to add? Like the roboticist. They would still lack the extra access and content but it would be at least a partial compromise, and could still add an extra workstation.

MilonPL commented:
Epic fail idiot didn't switch accounts.

I do think that having a separation between the "medbay chemist" and a pharmacist as a more crew-oriented role is a great idea, seeing how many chemists only focus on medbay and making meds for them and get upset whenever anyone asks for anything, the chemmasters quickly become full of random chemicals and can't be used for pills, etc.
Seeing as we already have shelves, and stuff, it might be a mapping issue of adding a "storefront" to the chem lab, even as a separate room with storage for pills. Templates for prescriptions and an alternative job title also sound like good ideas and content that could be implemented without making it the pharmacist a separate job.

Bribrooo commented:
> Epic fail idiot didn't switch accounts.
> 
> I do think that having a separation between the "medbay chemist" and a pharmacist as a more crew-oriented role is a great idea, seeing how many chemists only focus on medbay and making meds for them and get upset whenever anyone asks for anything, the chemmasters quickly become full of random chemicals and can't be used for pills, etc. Seeing as we already have shelves, and stuff, it might be a mapping issue of adding a "storefront" to the chem lab, even as a separate room with storage for pills. Templates for prescriptions and an alternative job title also sound like good ideas and content that could be implemented without making it the pharmacist a separate job.

I'd insist on having the pharmacist job implemented as such because there is no organization between chemists except at the start of the shift where they're like "Oh I'm going to make all of the bicar-related chems".
The Pharmacist, on the other hand, is the "crew's chemist" and basically could handle all the little needs of individuals, especially ones that you could consider "annoying" while trying to help out either the medical department, the epistemic, or any other department.
There have been many conversations on the pharmacist in the Discord's general (the audience's always right) and I truly believe that it would be a great addition despite it being a glorified chemist, at the service of the crew, and in charge of prescriptions that could get added too, that any doctor could hand to patients.

Bribrooo committed:
Resolving conflicts

Bribrooo committed:
Merge branch 'pharmacology' of https://github.com/Bribrooo/Delta-V-HA into pharmacology

Bribrooo committed:
Update chem_bag_pharmacist.yml

Bribrooo committed:
resolves conflict

Bribrooo committed:
AURGH

Bribrooo committed:
Moving files

Bribrooo committed:
Merge branch 'master' into pharmacology

Signed-off-by: BriBrooo <45323530+Bribrooo@users.noreply.github.com>

Bribrooo committed:
WORKS (should)

Bribrooo committed:
fixing yaml (?)

Bribrooo committed:
Merge branch 'master' into pharmacology

Bribrooo committed:
Update pharmacist.yml

Bribrooo committed:
Update Pharmacist.xml

Bribrooo committed:
Update pharmachemicals.yml

Bribrooo committed:
Update pharmachemicals.yml

Bribrooo committed:
Update pharmachemicals.yml

Bribrooo committed:
Update pharmachemicals.yml

Bribrooo committed:
Merge branch 'master' into pharmacology

Bribrooo committed:
Update pharmachemicals.yml

Bribrooo commented:
IM THIS FUCKING CLOSE TO EXPLODE EVERYTHING IN MY VICINITY

Bribrooo committed:
Merge branch 'pharmacology' of https://github.com/Bribrooo/Delta-V-HA into pharmacology

Bribrooo committed:
Update pharmachemicals.yml

Bribrooo committed:
Update pharmachemicals.yml

Bribrooo committed:
Update pharmachemicals.yml

Bribrooo committed:
Update pharmachemicals.yml

Bribrooo committed:
Update pharmachemicals.yml

Bribrooo committed:
Merge branch 'master' into pharmacology

Bribrooo committed:
Merge branch 'master' into pharmacology

Bribrooo committed:
Merge branch 'master' into pharmacology

Bribrooo committed:
Merge branch 'master' into pharmacology

Bribrooo committed:
updates prescription paper

Bribrooo committed:
Merge branch 'pharmacology' of https://github.com/Bribrooo/Delta-V-HA into pharmacology

Bribrooo committed:
Merge branch 'master' into pharmacology

Samsterious commented on Content.Shared/Access/Components/IdCardConsoleComponent.cs:
```diff
@@ -74,6 +74,7 @@ public WriteToTargetIdMessage(string fullName, string jobTitle, List<ProtoId<Acc
         "Mime", // DeltaV - Add Mime access
         "Musician", // DeltaV - Add Musician access
         "Paramedic", // DeltaV - Add Paramedic access
+        "Pharmacist", // DeltaV - Add Pharmacist access
```
The Pharmacist and the Psychologist have quite a lot of overlap, and I think they should be kept under the same access. Otherwise, for the 90% of stations that will not have them, this will be a button that does not do much taking up space in the console.

I might suggest something like "Psychiatry" as a shared access, renaming the existing Psychologist access and dropping Pharmacist entirely.

Bribrooo replied:
I would argue that Zookeeper has its own access.

> `89 90 "Zookeeper",  // DeltaV - Add Zookeeper access`

Samsterious commented on Resources/Locale/en-US/deltav/prototypes/access/accesses.ftl:
```diff
@@ -5,3 +5,4 @@ id-card-access-level-prosecutor = Prosecutor
 id-card-access-level-clerk = Clerk
 id-card-access-level-justice = Justice
 id-card-access-level-corpsman = Corpsman
+id-card-access-level-pharmacist = Pharmacist
```
Similar to above, dropping this and renaming the existing Psychology access to something like "Psychiatry".

Samsterious commented on Resources/Locale/en-US/deltav/reagents/pharmacymsg.ftl:
```diff
@@ -0,0 +1,42 @@
+reagent-effect-asparatine-overdose2 = It feels like you're getting stabbed in the guts.
+reagent-effect-asparatine-overdose3 = It is digging inside you.
+
+reagent-effect-asparatine-naguuprofen-mix1 = The Asparatine and Naguuprofen weren't a good idea...
```
If this was not copy-pasted directly from CD, we might avoid ick-ock. The character who ingested the medicine does not necessarily know that they have taken these drugs.

Samsterious commented on Resources/Prototypes/Access/medical.yml:
```diff
@@ -21,4 +21,5 @@
   - Medical
   - Chemistry
   - Paramedic # DeltaV - Add Paramedic access
+  - Pharmacist # DeltaV - Add Pharmacist access
```
More access references. ~~Pharmacist~~, Psychologist -> "Psychiatry", etc.

Samsterious commented on Resources/Prototypes/Access/misc.yml:
```diff
@@ -36,6 +36,7 @@
   - Orders # DeltaV - Orders, see Resources/Prototypes/DeltaV/Access/cargo.yml
   - Mantis # DeltaV - Psionic Mantis, see Resources/Prototypes/DeltaV/Access/epistemics.yml
   - Paramedic # DeltaV - Add Paramedic access
+  - Pharmacist # DeltaV - Add Pharmacist access
```
More access references. ~~Pharmacist~~, Psychologist -> "Psychiatry", etc.

Samsterious commented on Resources/Prototypes/DeltaV/Access/medical.yml:
```diff
@@ -1,3 +1,8 @@
   name: id-card-access-level-psychologist
+
+- type: accessLevel
+  id: Pharmacist
```
More access references. ~~Pharmacist~~, Psychologist -> "Psychiatry", etc.

Samsterious commented on Resources/Prototypes/DeltaV/Access/misc.yml:
```diff
@@ -40,6 +40,7 @@
     - Mantis
     - Paramedic
     - Psychologist
+    - Pharmacist
```
More access references. ~~Pharmacist~~, Psychologist -> "Psychiatry", etc.

Samsterious commented on Resources/Prototypes/DeltaV/Entities/Clothing/OuterClothing/coats.yml:
```diff
@@ -119,3 +119,17 @@
     coefficient: 0.1
 
+- type: entity
+  parent: ClothingOuterStorageFoldableBase
```
Perhaps this should be parented to Chemist's Winter Coat and inherit its protection stats, if it has any?

Bribrooo replied:
The protection stats are already there, and the parent needs to be foldable, like any lab coat.

Samsterious commented on Resources/Prototypes/DeltaV/Entities/Clothing/OuterClothing/coats.yml:
```diff
@@ -119,3 +119,17 @@
+  - type: Armor
+    modifiers:
+      coefficients:
+        Caustic: 0.75
```
Wow, that is a lot! Is this what Chemists get from their labcoats?

Bribrooo replied:
This is indeed what they get from their labcoats.

Samsterious commented on Resources/Prototypes/DeltaV/Entities/Objects/Devices/Electronics/door_access.yml:
```diff
@@ -125,3 +125,11 @@
+  suffix: Pharmacist, Locked
+  components:
+  - type: AccessReader
+    access: [["Pharmacist"]]
```
More access references. ~~Pharmacist~~, Psychologist -> "Psychiatry", etc.

Samsterious commented on Resources/Prototypes/DeltaV/Entities/Objects/Devices/pda.yml:
```diff
@@ -38,6 +38,44 @@
+  - type: Icon
+    sprite: DeltaV/Objects/Devices/pda.rsi
+    state: pda-pharmacist
+  - type: HealthAnalyzer
```
Psychologists do not get this, and the Pharmacist is even further removed from emergency medicine than Chemists. I would suggest omitting this or, alternatively, we decide that all Medical staff get health analyzers, period.

Bribrooo replied:
Removed

Samsterious commented on Resources/Prototypes/DeltaV/Entities/Structures/Doors/Airlocks/access.yml:
```diff
@@ -238,6 +238,15 @@
+- type: entity
+  parent: AirlockMedical
+  id: AirlockPharmacistLocked
+  suffix: Pharmacist, Locked
```
More access references. ~~Pharmacist~~, Psychologist -> "Psychiatry", etc.

Samsterious commented on Resources/Prototypes/DeltaV/Entities/Structures/Doors/Windoors/windoor.yml:
```diff
@@ -162,3 +162,12 @@
+- type: entity
+  parent: WindoorSecure
+  id: WindoorSecurePharmacistLocked
+  suffix: Pharmacist, Locked
```
More access references. ~~Pharmacist~~, Psychologist -> "Psychiatry", etc.

Samsterious commented on Resources/Prototypes/DeltaV/Entities/Structures/Storage/Closets/Lockers/lockers.yml:
```diff
@@ -23,3 +23,16 @@
+    stateDoorOpen: med_open
+    stateDoorClosed: pharmacy_door
+  - type: AccessReader
+    access: [["Pharmacist"]]
```
More access references. ~~Pharmacist~~, Psychologist -> "Psychiatry", etc.

Samsterious commented on Resources/Prototypes/DeltaV/Roles/Jobs/Medical/pharmacist.yml:
```diff
@@ -0,0 +1,32 @@
+  access:
+  - Medical
+  - Chemistry
+  - Pharmacist
```
More access references. ~~Pharmacist~~, Psychologist -> "Psychiatry", etc.

Samsterious commented on Resources/Prototypes/Entities/Objects/Tools/access_configurator.yml:
```diff
@@ -50,6 +50,7 @@
       - Mime #Delta V: Add Mime Access
       - Musician #Delta V: Add Musician Access
       - Paramedic # Delta V - adds Paramedic access
+      - Pharmacist # Delta V - adds Pharmacist access
```
More access references. ~~Pharmacist~~, Psychologist -> "Psychiatry", etc.

Samsterious commented on Resources/Prototypes/Roles/Jobs/departments.yml:
```diff
@@ -79,6 +79,7 @@
   - Psychologist
   - Paramedic
   - MedicalBorg  # Delta V - Medical Borg, see Resources/Prototypes/DeltaV/Roles/Jobs/Medical/medicalborg.yml
+  - Pharmacist # Delta V - Pharmacist, see Ressources/Prototypes/DeltaV/Roles/Jobs/Medical/pharmacist.yml
```
After discussion with Velcro, it sounds like this role will be used as a station-specific role, so it should be marked accordingly.

Samsterious commented on Resources/ServerInfo/Guidebook/DeltaV/Medical/Pharmacist.xml:
```diff
@@ -0,0 +1,40 @@
+<Document>
+# Pharmacist
+The "crème de la crème" in terms of knowledge in chemistry, the Pharmacist is the best at its domain! Whether they studied at Interdyne Pharmaceutics or the Andromeda Initiative Laboratories, their valuable knowledge is to be respected.
```
Let us not insert unintelligible lore babble into the guidebook. Nobody knows what these companies are; they have not been documented. A pharmacist is also not the Super Chemist, and I do not want players to misunderstand our intentions here.

Samsterious reviewed (changes requested):
Direction is ready to approve this PR after the requested changes have been made.

TadJohnson00 commented:
Is this still being worked on @Bribrooo?

Bribrooo commented:
Yeah just busy atm

TadJohnson00 commented:
All good! Take your time

Bribrooo committed:
Trying to fix conflicts

Bribrooo committed:
Merge branch 'master' into pharmacology

Signed-off-by: BriBrooo <45323530+Bribrooo@users.noreply.github.com>

Bribrooo committed:
Update rubber_stamp.yml

Bribrooo committed:
Merge branch 'pharmacology' of https://github.com/Bribrooo/Delta-V-HA into pharmacology

Bribrooo committed:
Resolving conversations

Bribrooo committed:
Merge branch 'master' into pharmacology

Bribrooo commented:
god strike me down because i am done

Bribrooo commented:
Actually I'm going to make the prescription paper a custom thing. . .

Bribrooo committed:
Merge branch 'master' into pharmacology

Bribrooo committed:
Merge branch 'master' into pharmacology

Signed-off-by: BriBrooo <45323530+Bribrooo@users.noreply.github.com>

Bribrooo commented:
nvm about the prescription thing... I'll add it later when I actually know what the hell I'm doing

Bribrooo committed:
Merge branch 'master' into pharmacology

Signed-off-by: BriBrooo <45323530+Bribrooo@users.noreply.github.com>

Bribrooo committed:
Update door_access.yml

Signed-off-by: BriBrooo <45323530+Bribrooo@users.noreply.github.com>

Toby222 reviewed (commented):
Is this still being worked on?
